lib/open_food_network/subscription_service.rb

```diff
@@ -0,0 +1,25 @@
+module OpenFoodNetwork
```

Comment:
I am having second thoughts about this namespace thing.
I think this namespace should be subscriptions only, we should drop OpenFoodNetwork. Also, I think this should be in app/services, with namespace becomes app/services/subscriptions. Just like most reports stuff in lib/open_food_network should be in app/services/reports (namespace should be reports). I am opening a thread on discourse to tackle this one.

Reply:
here's the new thread: https://community.openfoodnetwork.org/t/location-of-services-in-the-codebase/1435

Reply:
this approach has been approved meanwhile. but maybe too much trouble to get it done as part of this PR?

Reply:
Working on this again. ;-) Moving it to `OrderManagement` now, @luisramos0.

Reply:
awesome!
are u not getting into a big pr? maybe do that in a follow up pr?

Reply:
Aha! Sorry yes, I forgot we don't have the `OrderManagement` engine yet because that still comes with the "Enterprise Fee Summary" changes. So moving to the engine will have to follow in this new tech debt issue: #3356

For now, I moved the service directly into `app/services/`, without `OpenFoodNetwork`.